galata/package.json

```
@@ -66,7 +66,7 @@
     "path": "~0.12.7",
     "systeminformation": "^5.8.6",
     "vega": "^5.20.0",
-    "vega-lite": "^5.1.0",
+    "vega-lite": "^5.6.1-next.1",
```

Looks like the final release is now available: https://github.com/vega/vega-lite/releases/tag/v5.6.1

```suggestion
    "vega-lite": "^5.6.1",
```